lightning/src/ln/outbound_payment.rs

```diff
@@ -1882,6 +1924,8 @@ impl_writeable_tlv_based_enum_upgradable!(PendingOutboundPayment,
 		(0, expiration, required),
 		(2, retry_strategy, required),
 		(4, max_total_routing_fee_msat, option),
+		(5, invoice_request, option),
+		(6, nonce, option),
```

Comment:
We follow the "it's ok to be odd" rule for our TLVs. If a downgraded node sees an even TLV it doesn't know how to interpret, then it will fail to deserialize it. So this needs to be odd.